nixos/tests/systemd-boot.nix

```diff
@@ -14,6 +14,11 @@ let
     boot.loader.efi.canTouchEfiVariables = true;
     environment.systemPackages = [ pkgs.efibootmgr ];
   };
+
+  common_xbootldr = recursiveUpdate common {
```

Comment:
Use the module system to do merging, not `recursiveUpdate`. Just add `imports = [ common ];` instead.

Reply:
Updated.